src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -65,6 +65,11 @@ public bool CanInline(MethodDesc caller, MethodDesc callee)
 
         public virtual MethodIL GetMethodIL(MethodDesc method)
         {
+            if (method is UnboxingMethodDesc unboxingMethodDesc)
```

Comment:
Could you unwrap this within JitInterface? The existence of `UnboxingMethodDesc` is private to the JitInterface - it should never escape it (plus it's nice for the other places that call this not to have to pay for the type check).

Reply:
Fixed in 2nd commit.